Optimizer/Minuit2/MinuitResult.hpp

```diff
@@ -9,35 +9,45 @@
 
 #include "boost/serialization/base_object.hpp"
 
+namespace ROOT {
+namespace Minuit2 {
+class MnUserParameterState;
+class FunctionMinimum;
+} // namespace Minuit2
+} // namespace ROOT
+
 namespace ComPWA {
 namespace Optimizer {
 namespace Minuit2 {
 
 struct MinuitResult : public FitResult {
-  bool CalcInterference;
-  bool IsValid;             // result valid
-  bool CovPosDef;           // covariance matrix pos.-def.
-  bool HasValidParameters;  // valid parameters
-  bool HasValidCov;         // valid covariance
-  bool HasAccCov;           // accurate covariance
-  bool HasReachedCallLimit; // call limit reached
-  bool EdmAboveMax;
-  bool HesseFailed;
-  double ErrorDef;
-  unsigned int NFcn;
-  double Edm; // estimated distance to minimum
+  MinuitResult() = default;
+  MinuitResult(const FitResult &Result,
+               const ROOT::Minuit2::FunctionMinimum &FMin);
+  bool IsValid = false;             // result valid
```

Comment:
Hmm, true. I agree a default constructor can be deleted here. I just needed the default constructor to make some of the tests working, for example `Tools/tests/SaveAndLoadFitResultsTest.cpp`. I will change that

Reply:
Ok, so I would need to do some modifications in the boost serialization, since this is the main problem here. https://www.boost.org/doc/libs/1_67_0/libs/serialization/doc/index.html (Non-Default Constructors section)

Reply:
I see. My main point was actually to remove the default values.

Reply:
But that is exactly what a default constructor does. It will just initialize everything with 0/false